src/test/java/org/codehaus/plexus/archiver/dir/DirectoryArchiverTest.java

```diff
@@ -0,0 +1,34 @@
+package org.codehaus.plexus.archiver.dir;
+
+import java.io.File;
+import java.io.IOException;
+import org.junit.Test;
+
+public class DirectoryArchiverTest {
+
+    private static final File CURRENT_DIR = new File(".");
+    private static final File DEST_DIR = new File("target/output/testCreateDirsForSymlink");
+    private static final File SYMLINK_FILE = new File(DEST_DIR, "testSymlink");
+
+    /**
+     * Test case for ISSUE-130
+     * @throws IOException
+     */
+    @Test
+    public void testSymlinkWithParentDirectory()
+        throws IOException
+    {
+        removeSymlink();
+        DirectoryArchiver archiver = new DirectoryArchiver();
+        archiver.addSymlink( SYMLINK_FILE.getPath(), CURRENT_DIR.getPath() );
+        archiver.setDestFile( CURRENT_DIR );
+        archiver.execute();
```

Comment:
Thanks a lot for the tests. But there is already existing tests for symlinks for dir archivers (sorry forgot to mention it) - org.codehaus.plexus.archiver.SymlinkTest#testSymlinkDirArchiver. Maybe it would be better if this test is added there - to the same method or maybe a new method inside the same test class. I could help with it if you want.

Reply:
Thanks for that, I didn't see the existing tests. That simplifies things.